cicecore/cicedyn/infrastructure/ice_domain.F90

```diff
@@ -242,16 +257,6 @@ subroutine init_domain_blocks
       !*** domain size zero or negative
       !***
       call abort_ice(subname//' ERROR: Invalid domain: size < 1', file=__FILE__, line=__LINE__) ! no domain
-   else if (nprocs /= get_num_procs()) then
-      !***
-      !*** input nprocs does not match system (eg MPI) request
```

Comment:
we could keep this comment, I think

Reply:
Isn't this clear from the ERROR message?